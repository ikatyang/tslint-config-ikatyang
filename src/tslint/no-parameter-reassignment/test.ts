// good

function good1(a = 1) {

}

// bad

function bad1(a) {
  a = a || 1;
}
