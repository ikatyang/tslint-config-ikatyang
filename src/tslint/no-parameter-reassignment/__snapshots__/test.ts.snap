// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

function good1(a = 1) {

}

// bad

function bad1(a) {
  a = a || 1;
  ~           [Reassigning parameter 'a' is forbidden.]
}

`;
